src/sentry/processing/backpressure/monitor.py

```diff
@@ -90,7 +90,13 @@ def check_service_health(services: Mapping[str, Service]) -> MutableMapping[str,
             for memory in check_service_memory(service):
                 if memory.percentage >= high_watermark:
                     reasons.append(memory)
-                logger.info("Checking node: %s:%s", memory.host, memory.port)
+
+                if memory.host is not None and "@" in memory.host:
+                    passwordless_host = memory.host.split("@")[1]
+                else:
+                    passwordless_host = memory.host
```

Comment:
Do we have tests for backpressure? Seems like a good thing to have tests for.

Reply:
Yep - https://github.com/getsentry/sentry/blob/533a82ed8219f16bed57c1e1bd70c0f74076845a/tests/sentry/processing/backpressure/test_monitoring.py